sros2/test/policies/add_two_ints.policy.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
minimal note: It would be good to not squash the commit renaming files before merging, to have a cleaner history.